content/posts/2025-plans.md

```diff
@@ -1,25 +1,28 @@
 ---
 title: "Open Ecoacoustics 2025 Plans"
-date: 2025-02-05T12:19:00+10:00
-draft: true
+date: 2025-02-06T12:19:00+10:00
+image: '/images/sensor-occupant.png'
 tags:
 - News
 ---
 I hope your 2025 is off to a great start. We at Open Ecoacoustics are excited to keep the community more updated on our activities and progress. This year, we're committed to providing more regular updates and sharing our work through these posts and on social media. 
-
 <!--more-->
+
 ## Follow Us on BlueSky and LinkedIn 
 
 We are excited to announce the launch of our accounts on [BlueSky](https://bsky.app/profile/openecoacoustics.bsky.social) and [LinkedIn](https://www.linkedin.com/company/open-ecoacoustics/). Follow us to stay in the loop! 
 
+_Cover image of this post and our social media profiles by Prof. Susan Fuller_
+
+## New Feature: Audio Verification Interface
+Last year we finished the first version of an audio verification component. This is an open-source component that can be embedded on any website. We are currently working on adding it to [Ecosounds](https://www.ecosounds.org/). Here is a quick gif to show you how it works:
+{{% figure src="images/verification-interface.gif" caption="Verification Interface." width="50%"%}}
 
 ## Looking Ahead to 2025  
+In addition to launching the Verification Interface tool, some of our other plans for 2025 include: 
 
-Some of our plans for 2025 include:  
 - Improving mapping visualisations for clearer data coverage views  
-- Continuing development of the Acoustic Workbench (The software that runs [A2O](https://acousticobservatory.org/) & [Ecosounds](https://www.ecosounds.org/))  
+- Continuing development of the Acoustic Workbench (The software that runs [A2O](https://acousticobservatory.org/) & [Ecosounds](https://www.ecosounds.org/)). Did you knpow Acoustic Workhbench is open source? See the source code in [GitHub](https://github.com/QutEcoacoustics/workbench)
 - Using our new verification interface for citizen science projects  
-- Engaging the community through user testing workshops with selected partners
 - Improving the [Make Your Own Recogniser](https://openecoacoustics.org/resources/lessons/make-your-own-recognizer/) workshop and training materials
-
-We also have plans for organising some additional training. Stay in the loop!
\ No newline at end of file
+- Engaging the community through user testing and additional training workshops. Expect us to post an update in the coming months!
```

Comment:
do we want to mention our other milestones? AAF, PIDs, etc??

also we should a call to action "let us know which of these you're most keen for! Do you want something else? Let us know that too! <link to contact form>"